sapi/phpdbg/phpdbg.c

```diff
@@ -1369,14 +1370,7 @@ int main(int argc, char **argv) /* {{{ */
 			if (show_help) {
 				phpdbg_do_help_cmd(exec);
 			} else if (show_version) {
-				phpdbg_out(
-					"phpdbg %s (built: %s %s)\nPHP %s, Copyright (c) The PHP Group\n%s",
-					PHPDBG_VERSION,
-					__DATE__,
-					__TIME__,
-					PHP_VERSION,
-					get_zend_version()
-				);
+				php_print_version(&phpdbg_sapi_module);
```

Comment:
From what I see phpdbg has got some abstraction and could potentially be used for other fd's (eplicitly setting io field) even though it seems it is currently just set for std streams. Not sure if it's a good idea to replace it though.

Reply:
This makes sense, a version that provides it as a string for the sake of `phpdbg_out` would be useful.